torch/testing/_internal/common_utils.py

```diff
@@ -1728,6 +1728,10 @@ def random_sparse_matrix(rows, columns, density=0.01, **kwargs):
     dtype = kwargs.get('dtype', torch.double)
     device = kwargs.get('device', 'cpu')
     singular = kwargs.get("singular", False)
+    values_domain = kwargs.get("domain", None)
```

Comment:
In the future we should also add tests to test_testing to ensure that random_sparse_matrix() (and make_tensor()) work as expected